docker admin: create company & settings update (#17705)
docker/docs: admin/organization/general-settings.md

1. Sign in to [Docker Admin](https://admin.docker.com){: target="_blank" rel="noopener" class="_"}.
2. In the left navigation, select your organization in the drop-down menu.
3. Under **Organization Settings**, select **General**.
4. Specify the organization information and select **Save**.

## Create a company

To create a new company:

1. Sign in to [Docker Admin](https://admin.docker.com){: target="_blank" rel="noopener" class="_"}.
2. In the left navigation, select your organization in the drop-down menu.
3. Under **Organization Settings**, select **General**.

> 4. Near the bottom of the page, select **Create a company**.

Reviewer: The only orgs I have currently are in a company already, would this be found under **Organization management**? If that's the case, maybe this would be more clear:
```suggestion
4. In the **Organization management** section, select **Create a company**.
```

Author: Yes. In Hub it's also under **Organization management**.
